appcoins-ads/build.gradle

```diff
@@ -46,12 +52,8 @@ dependencies {
   testImplementation lib.mockito_core
 
   androidTestImplementation lib.android_test_runner
-
-  api lib.retrofit2
   api lib.tray
-  api lib.reactivenetwork_rx2
-
-  api project(path: ':appcoins-contract-proxy')
+  api project(path: ':appcoins_net')
```

Comment:
Can we improve this name, since the module is generic, does it needs to be appcoins related? And "net" part, can it be more explicit.